### PR TITLE
Use querySelectorAll instead of recursive search

### PR DESCRIPTION
**Pull request by cicloon**

### CODE REVIEW

**Comment by patriciagao** [comment]
:+1:

**Comment by ciscou** [comment]
Awesome.

Clearly the plugin wasn't made to have tons of DOM elements inside the map container. But it was clearly doable :)

Well done!

**Comment by ropa03** [comment]
As I told you. OSOM!

File: www/googlemaps-cdv-plugin.js [hunk 1]
```diff
@@ -2607,31 +2607,26 @@ window.addEventListener("orientationchange", function() {
...
-                    }
-                }
-            }
-            node = node.nextSibling;
+    var style, displayCSS, opacityCSS, visibilityCSS, node;
+
+    var allElements = Array.prototype.slice.call(document.querySelectorAll('*'));
+    var notClickableElements = Array.prototype.slice.call(document.querySelectorAll('[data-clickable="false"]'));
+    notClickableElements.push(root)
+    var clickableElements = allElements.filter(function(i) {return notClickableElements.indexOf(i) < 0;});
```
Comment (amuino) [inline www/googlemaps-cdv-plugin.js]: Probably stupid but… wouldn't it be faster finding clickable elements by negating the selector used for "not clickable"?
`querySelectorAll(':not([data-clickable="false"])')`

Comment (cicloon) [inline www/googlemaps-cdv-plugin.js]: You are right, I though that couldn't be done with querySelectorAll.

Comment (amuino) [inline www/googlemaps-cdv-plugin.js]: Well, test it first… I'm a backend guy now :innocent: But I think it should be safe (at least with CrossWalk)

Comment (cicloon) [inline www/googlemaps-cdv-plugin.js]: It works like a charm, and I noticed a little increase in performance too.

**Pull request merged**